linkatos/parser.py

```diff
@@ -27,4 +27,17 @@ def parse_reaction_added(event):
         'user': event['user'],
         'to_user': event['item_user']
     }
+
     return reaction
+
+
+def parse_list_request(event):
+    if not message.is_list_request(event['text']):
+        return {'type': None}
```

Comment:
Because afterwards (line 41 of `activities.py` I assume the 'type' field of the outcome object exists if the object is `None` it gives me an error.
The alternative options I can think of are to either first filter for a `None` object or to check if 'type' is in the object.
What would you do?